README.md

```diff
@@ -60,22 +76,53 @@ val value by intSort
 val expr = (array.select(index - 1.expr) lt value) and
         (array.select(index + 1.expr) gt value)
 ```
-Check out our [example project](examples) for more complicated examples.
 
-### Solver agnostic SMT formula representation
-KSMT provides a solver-independent formula representation
-with back and forth transformations to the solver's native representation.
-Such representation allows introspection of formulas and transformation of formulas 
-independent of the solver.
+With this DSL, you can
```

Comment:
We don't provide DSL for formula manipulation. Our DSL is only suitable for the formula creation.